dotcom-rendering/src/web/components/Card/components/CardLayout.tsx

```diff
@@ -37,8 +37,8 @@ const decideWidth = (minWidthInPixels?: number) => {
 };
 
 const decidePosition = (
-	imagePosition: ImagePositionType,
-	imagePositionOnMobile: ImagePositionType,
+	imagePosition?: ImagePositionType,
+	imagePositionOnMobile?: ImagePositionType,
```

Comment:
Not sure I understand why these two are changing to optional. In the `Props` types they remain non-optional: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/Card/components/CardLayout.tsx#L7-L8

And in `Card.tsx` above I see this:
<img width="827" alt="image" src="https://user-images.githubusercontent.com/19683595/192577235-f313f16d-6d37-463c-a8f3-7889c0d8d438.png">

Reply:
You are right, I have now fixed this in [b27392f](https://github.com/guardian/dotcom-rendering/pull/6066/commits/b27392f7ff6593bfc479bea0b1c7dfdece1d1d4d)